src/main/java/mate/jdbc/dao/impl/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,131 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    private static final String INSERT_MANUFACTURER_REQUEST
+            = "INSERT INTO manufacturers(name, country) values(?,?);";
+    private static final String GET_ALL_MANUFACTURERS_REQUEST
+            = "SELECT * FROM manufacturers WHERE is_deleted = false";
+    private static final String UPDATE_MANUFACTURER_REQUEST
+            = "UPDATE manufacturers SET name=?,country=? WHERE id=?";
+    private static final String DELETE_MANUFACTURER_REQUEST
+            = "UPDATE manufacturers SET is_deleted = true WHERE id = ?";
+    private static final String GET_MANUFACTURER_REQUEST
+            = "SELECT * FROM manufacturers WHERE is_deleted = false AND id = ?";
+    private static final String CANT_GET_ALL_MESSAGE = "Can't get all manufacturers from DB.";
+    private static final String CANT_CREATE_MESSAGE = "Can't insert manufacturer to DB.";
+    private static final String CANT_UPDATE_MESSAGE = "Can't update manufacturer in DB.";
+    private static final String CANT_DELETE_MESSAGE = "Can't delete manufacturer from DB.";
+    private static final String CANT_GET_MESSAGE = "Can't get manufacturer by id from DB.";
+    private static final String NAME = "name";
+    private static final String COUNTRY = "country";
+    private static final String ID = "id";
+
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> manufacturerList = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                 Statement getAllManufacturersStatement = connection.createStatement();) {
+            ResultSet resultSet = getAllManufacturersStatement
+                    .executeQuery(GET_ALL_MANUFACTURERS_REQUEST);
+            while (resultSet.next()) {
+                String name = resultSet.getString(NAME);
+                String country = resultSet.getString(COUNTRY);
+                Long id = resultSet.getObject(ID, Long.class);
+                Manufacturer manufacturer = new Manufacturer();
+                manufacturer.setId(id);
+                manufacturer.setName(name);
+                manufacturer.setCountry(country);
+                manufacturerList.add(manufacturer);
```

Comment:
same for this part, create separate method to work with ResulSet

Reply:
fixed